bql/semantic/semantic_test.go

```diff
@@ -324,6 +349,27 @@ func TestIsEmptyClause(t *testing.T) {
 
 }
 
+func TestIsEmptyFilterClause(t *testing.T) {
+	testTable := []struct {
+		in  *FilterClause
+		out bool
+	}{
+		{
+			in:  &FilterClause{},
+			out: true,
+		},
+		{
+			in:  &FilterClause{Operation: "latest", Binding: "?p"},
+			out: false,
+		},
+	}
+	for _, entry := range testTable {
+		if got, want := entry.in.IsEmpty(), entry.out; got != want {
```

Comment:
if entry.in.IsEmpty() != entry.want {
?

Reply:
Here I was just following the pattern set in this same file by a very similar test function `TestIsEmptyClause`, but I prefer it the way you suggested as well. I have just changed it then.